runbooks/workflow/creating-maintaining-littlepay-data-syncs.md

```diff
@@ -0,0 +1,51 @@
+# Creating and Maintaining Littlepay Data Syncs
+
+> Some of this is taken from the provided Littlepay documentation, with Cal-ITP specific content added.
+
+We access Littlepay data via AWS S3 file storage, and use Airflow-orchestrated sync tasks to bring that data into Cal-ITP's GCS buckets and then into our data warehouse.
```

Comment:
Can we link to the [Cal-ITP data ingest diagram](https://docs.calitp.org/data-infra/architecture/data.html) here?

Reply:
Resolved in [0049f24](https://github.com/cal-itp/data-infra/pull/3019/commits/0049f24032f29d2a243d2db04fe472dcd4067ae7)